lib/apiForSheets.js

```diff
@@ -113,26 +113,15 @@ module.exports.checkClaSheet = async function(auth) {
       return;
     }
 
-    var pullRequestsPromiseObj = await context.github.pullRequests.getAll(
-      context.repo({per_page: 40}));
-
-    var arrayOfOpenPullRequests = pullRequestsPromiseObj.data;
-
-    for (var indexOfPullRequest in arrayOfOpenPullRequests) {
-      var pullRequestNumber = arrayOfOpenPullRequests[
-        indexOfPullRequest].number;
-      var pullRequestDetailsPromiseObj = await context.github.pullRequests.get(
-        context.repo({number: pullRequestNumber}));
-
-      var pullRequestDetails = pullRequestDetailsPromiseObj.data;
+      var pullRequestNumber = context.payload.pull_request.number;
```

Comment:
Deindent this and the following lines by 2.

Reply:
Done, thanks!